ios/RNSScreenStackHeaderConfig.m

```diff
@@ -146,11 +147,11 @@ + (void)setAnimatedConfig:(UIViewController *)vc withConfig:(RNSScreenStackHeade
         attrs[NSForegroundColorAttributeName] = config.titleColor;
       }
 
-      CGFloat size = config.titleFontSize ? [config.titleFontSize floatValue] : 17;
+      NSNumber *size = config.titleFontSize ? config.titleFontSize : @17;
```

Comment:
If you were to rewrite it to `NSNumber *size` it could be written more shorter with the Elvis operator:

`NSNumber *size = config.titleFontSize ?: @17;`

but from my point of view it could as well be left as it was before and we could wrap the `CGFloat size` into an `NSNumber *` in the positive `if` clause.